README.md

```diff
@@ -734,9 +734,9 @@ All statements above will return false if used with `===`
 
 <br/><br/>
 
-## ![✔] 5.19. Install your packages with `npm ci`
+## ![✔] 5.19. Install your packages with `npm ci` 
 
-**TL;DR:** You have to be sure that production code uses the exact version of the packages you have tested it with. Run `npm ci` to do a clean install of your dependencies matching package.json and package-lock.json.
+**TL;DR:** You have to be sure that production code uses the exact version of the packages you have tested it with. Run `npm ci` to do a clean install of your dependencies matching package.json and package-lock.json. Using this command is recommended in automated environments such as continuous integration pipelines. It is stricter by throwing an error if the versions in package.json and package-lock.json do not match. Since it can be significantly faster than `npm install`, it can also save you CI pipeline minutes. See [here](https://docs.npmjs.com/cli/ci.html#description) for more info.
```

Comment:
Cool! Can you also add these pertinent info to the `docker/installpackageswithnpmci.md` page?

Reply:
I haven't found this file. Do you mean [`docker/install-for-production.md`](https://github.com/bttger/nodebestpractices/blob/master/sections/docker/install-for-production.md)? This section already mentions the `npm ci`command and also quotes the npm docs at the bottom.

Reply:
Ah I found it, it was under the `production` directory. Going to shorten the TL;DR and put it into the "more info" page.